docs/Installation.md

```diff
@@ -276,7 +283,17 @@ See appendices for [MariaDB][MariaDB instructions for Debian] and
 [PostgreSQL][PostgreSQL instructions for Debian].
 
 
-### 4.3 Service configuration and startup (Debian/Ubuntu)
+### 4.3 Database configuration (Debian/Ubuntu)
+
+Finally initialize the database:
+
+```sh
+cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
+sudo -u zonemaster perl create_db.pl
+```

```

Comment:
Suggestion to make it easier to just copy and paste:

1. Make `create_db.pl` executable
2. Make the command a one-liner:
``` sh
sudo -u zonemaster $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')/create_db.pl
```

Reply:
done